isleconfig.py

```diff
@@ -13,6 +13,7 @@
                        "no_riskmodels": 2,
                        "riskmodel_inaccuracy_parameter": 2,  # values >=1; inaccuracy higher with higher values
                        "riskmodel_margin_of_safety": 2,  # values >=1; factor of additional liquidity beyond value at risk
+                       "margin_increase": 0.6, # This parameter modifies the margin of safety depending on the number of risks models available in the market. When is 0 all risk models have the same margin of safety.
```

Comment:
I would suggest to put the default of this to zero. (Precisely because the implementation of this in `metainsuranceorg.py` is unrealistic and difficult to defend.)

Reply:
Done!